deploy_polygon/05_catalyst/07_setup_asset_upgrader.ts

```diff
@@ -17,7 +18,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   if (!isAssetUpgraderSandSuperOperator) {
     await execute(
```

Comment:
We're missing a catch around the execute (because we don't have the private key of sandAdmin when running this so we catch the call to output the tx to be run by sandAdmin manually)
```
await catchUnknownSigner(
(...)
)
```